trio/_core/_multierror.py

```diff
@@ -341,12 +344,14 @@ def concat_tb(head, tail):
 
 ################################################################
 # MultiError traceback formatting
+#
+# What follows is terrible, terrible monkey patching of
+# traceback.TracebackException to add support for handling
+# MultiErrors
 ################################################################
 
 
-# format_exception's semantics for limit= are odd: they apply separately to
-# each traceback. I'm not sure how much sense this makes, but we copy it
-# anyway.
+@deprecated("0.2.0", issue=305, instead="traceback.format_exception")
```

Comment:
I wasn't sure what issue number to use for this. I used #305 but it doesn't really address the deprecation. Should I make an issue specifically for this?

Reply:
You can also use the PR number if it makes more sense - which in this case it probably does, I agree.

Reply:
(This works because github doesn't fully distinguish between PRs and issues – they use the same number space, and the URL for "issue 347" redirects to the PR page if 347 is a PR.)